include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -31,7 +31,7 @@
 #include <span>
 #endif
 #if defined(_MDSPAN_USE_CONCEPTS) && MDSPAN_HAS_CXX_20
-#include<concepts>
+#include <type_traits>  // for std::is_same_v
 #endif
```

Comment:
`trait_backports.hpp` (included above) already includes `<type_traits>` unconditionally, so we don't need special logic to include it again here.  Also, would you consider only doing the work-around if the `<concepts>` header is not available, i.e., if the feature test macro `__cpp_lib_concepts` is not defined?

```c++
#if defined(_MDSPAN_USE_CONCEPTS) && MDSPAN_HAS_CXX_20 && defined(__cpp_lib_concepts)
#  include <concepts>
#endif
```